libs/s3/s3_multipart_test_lib.py

```diff
@@ -175,35 +153,24 @@ def upload_parts(self,
                     LOGGER.debug("Part : %s", str(part))
                     parts.append({"PartNumber": i, "ETag": part["ETag"]})
                     uploaded_bytes += len(data)
-                    LOGGER.debug(
-                        "{0} of {1} uploaded ({2:.2f}%)".format(
-                            uploaded_bytes,
-                            multipart_obj_size *
-                            1048576,
-                            cal_percent(
-                                uploaded_bytes,
-                                multipart_obj_size *
-                                1048576)))
+                    LOGGER.debug("%s of %s uploaded %.2f%%", uploaded_bytes, multipart_obj_size *
+                                 1048576, cal_percent(uploaded_bytes, multipart_obj_size * 1048576))
                     i += 1
             LOGGER.info(parts)
 
             return True, parts
         except (ClientError, Exception) as error:
-            LOGGER.exception("Error in %s: %s",
-                             S3MultipartTestLib.upload_parts.__name__,
-                             error)
-            raise CTException(err.S3_CLIENT_ERROR, error.args[0])
+            LOGGER.exception(ERR_MSG, S3MultipartTestLib.upload_parts.__name__, error)
+            raise CTException(err.S3_CLIENT_ERROR, error.args[0]) from error
 
     # pylint: disable-msg=too-many-locals
-    def upload_precalculated_parts(self,
-                                   upload_id: int = None,
-                                   bucket_name: str = None,
-                                   object_name: str = None,
-                                   **kwargs) -> tuple:
+    def upload_precalculated_parts(self, upload_id: int = None, bucket_name: str = None,
+                                   object_name: str = None, **kwargs) -> tuple:
         """
-        Upload specified/precalculated part sizes for a specific multipart upload ID one part
-        at a time.
+        Upload precalculated parts.
 
+        Upload specified/precalculated part sizes for a specific multipart uploadID one part at
+        a time.
```

Comment:
Let's keep either line 170 or 172-173. Repetition in the docstring seems unnecessary